ui/app/src/utils/itemActions.ts

```diff
@@ -313,12 +313,14 @@ export function generateSingleItemOptions(item: DetailedItem, permissions: Looku
         if (deleteItem) {
           _optionsA.push(menuOptions.delete);
         }
+        _optionsA.push(menuOptions.cut);
+        _optionsA.push(menuOptions.copy);
         if (type === 'taxonomy' || type === 'component') {
```

Comment:
taxonomy is not an existing system type

Reply:
this was already changed on the PR https://github.com/craftercms/studio-ui/pull/2130